JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv

```diff
@@ -22,7 +22,7 @@ AnnualHeatingConsumption,Class,HeatingConsumption,IS-A,,,,,https://www.theworlda
 AnnualHeatingConsumption,Class,AnnualThermalConsumption,IS-A,,,,,https://www.theworldavatar.com/kg/ontoubemmp/,Annual heating consumption
 GridConsumption,Class,EnergyConsumption,IS-A,,,,The total grid electricity energy consumption,https://www.theworldavatar.com/kg/ontoubemmp/,Grid consumption
 AnnualGridConsumption,Class,GridConsumption,IS-A,,,,,https://www.theworldavatar.com/kg/ontoubemmp/,Annual grid consumption
-consumesEnergy,Object Property,,,https://www.theworldavatar.com/kg/ontodevice/Device UNION https://w3id.org/bot#Zone UNION http://www.w3.org/ns/ssn/System,EnergyConsumption,,"A relationship between a device, zone or system and its consumed energy",https://www.theworldavatar.com/kg/ontoubemmp/,consumes energy
+consumesEnergy,Object Property,,,https://www.theworldavatar.com/kg/ontodevice/Device UNION https://w3id.org/bot#Zone UNION http://www.theworldavatar.com/ontology/ontocape/upper_level/technical_system.owl#TechnicalSystem,EnergyConsumption,,"A relationship between a device, zone or technical system and its consumed energy",https://www.theworldavatar.com/kg/ontoubemmp/,consumes energy
```

Comment:
Oh hmmm it's because since we are not using ssn:System, I thought it will be appropriate to remove it and only include it when we do decide to utilize that concept.